es6/index.d.ts

```diff
@@ -1,2 +1,2 @@
-declare const equal: (a: any, b: any) => boolean;
-export = equal;
+declare function equal<T>(actual: any, expected: T): actual is T
```

Comment:
Super nit, maybe

```suggestion
declare function equal<T>(actual: unknown, expected: T): actual is T
```

?

Reply:
The guideline I've followed, and I've seen other use, is to accept `any` and return `unknown`, so I think that this should stay as `any`. Can't find a good link for that at the moment though...